types/@ember/runloop.d.ts

```diff
@@ -0,0 +1,339 @@
+/**
+ * Temporarily copying these types into @ember/test-helpers until
```

Comment:
Would you mind adding something like BEGIN / END comments so we can tell _which_ parts can be deleted? Or is it that we delete everything?

Reply:
Ya, we delete the whole shebang. Specifically, it will be replaced with an extension of the `RunNamespace` from `@types/ember__runloop` (https://github.com/DefinitelyTyped/DefinitelyTyped/pull/32970/files#diff-b87fda3978af44033e6da75ab82d1bc9R9).

Reply:
👏